examples/lib/stories/effects/move_effect_example.dart

```diff
@@ -2,8 +2,8 @@ import 'dart:math';
 
 import 'package:flame/components.dart';
 import 'package:flame/game.dart';
+import 'package:flame/src/effects2/controllers/standard_controller.dart'; // ignore: implementation_imports
 import 'package:flame/src/effects2/move_effect.dart'; // ignore: implementation_imports
```

Comment:
should we just export these into `flame/effects.dart`? I am always weary of "hiding" stuff from the user specially when it seems there is a use case for them wanting to use it below.

Reply:
Check #1157 @luanpotter :)